Ddaangn/src/main/java/com/dev/ddaangn/user/controller/TempController.java

```diff
@@ -0,0 +1,67 @@
+package com.dev.ddaangn.user.controller;
+
+import com.dev.ddaangn.common.api.ApiResponse;
+import com.dev.ddaangn.user.config.auth.LoginUser;
+import com.dev.ddaangn.user.config.auth.dto.SessionUser;
+import com.dev.ddaangn.user.dto.ListResponseDto;
+import com.dev.ddaangn.user.dto.TempSaveResponseDto;
+import com.dev.ddaangn.user.service.TempService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value ="/api/v1")
```

Comment:
```suggestion
@RequestMapping(value ="/api/v1/temperature")
```
로 공통사항을 합칠 수 있을 것 같습니다!